hammer/technology/sky130/__init__.py

```diff
@@ -178,25 +183,15 @@ def openram_sram_names() -> List[str]:
             "sky130_sram_1kbyte_1rw1r_8x1024_8",
             "sky130_sram_2kbyte_1rw1r_32x512_8"
         ]
-
+    
     @staticmethod
     def sky130_sram_names() -> List[str]:
-        return [
-            "sramgen_sram_32x32m2w8_replica_v1",
-            "sramgen_sram_64x32m4w32_replica_v1",
-            "sramgen_sram_512x32m4w8_replica_v1",
-            "sramgen_sram_1024x32m8w8_replica_v1",
-            "sramgen_sram_1024x32m8w32_replica_v1",
-            "sramgen_sram_1024x64m8w32_replica_v1",
-            "sramgen_sram_2048x32m8w8_replica_v1",
-            "sramgen_sram_4096x32m8w8_replica_v1",
-
-            "sky130_sram_1kbyte_1rw1r_32x256_8",
-            "sky130_sram_1kbyte_1rw1r_8x1024_8",
-            "sky130_sram_2kbyte_1rw1r_32x512_8",
-            "sky130_sram_4kbyte_1rw1r_32x1024_8",
-            "sky130_sram_8kbyte_1rw1r_32x2048_8"
-        ]
+        sky130_sram_names = []
+        with open(f"{Path(__file__).parent.resolve()}/sram-cache.json",'r') as f:
```

Comment:
This isn't robust when Hammer is installed as a package. [Recommend using `importlib` instead](https://github.com/ucb-bar/hammer/blob/2522bdbcfcacfbeb475769c3c26de12b2e7addde/doc/Hammer-Basics/Migration-Guide.md?plain=1#L137-L156).